src/opensynth/datasets/low_carbon_london/get_data.py

```diff
@@ -3,47 +3,155 @@
 
 import logging
 from pathlib import Path
+from typing import Optional
 
 from opensynth.datasets import datasets_utils
-from opensynth.datasets.low_carbon_london import (
-    preprocess_lcl,
-    split_households,
-)
-
-LCL_URL = "https://data.london.gov.uk/download/smartmeter-energy-use-data-in-london-households/3527bf39-d93e-4071-8451-df2ade1ea4f2/LCL-FullData.zip"  # noqa
-FILE_NAME = Path("data/raw/lcl_full_data.zip")  # noqa
-CSV_FILE_NAME = Path("data/raw/CC_LCL-FullData.csv")  # noqa
+from opensynth.datasets.low_carbon_london import split_households
+from src.opensynth.datasets.low_carbon_london import preprocess_lcl
```

Comment:
Should this be a relative import, not an absolute import? I'm confused why the import on the line above is different to this one? 
```suggestion
from opensynth.datasets.low_carbon_london import preprocess_lcl
```